portal/server/controllers/dashboard/deals/deals-filters-query.js

```diff
@@ -94,15 +88,24 @@ const dashboardDealsFiltersQuery = (createdByYou, filters, user) => {
        *    { $or: [{dealType: 'GEF}, {dealType: 'BSS'}] },
        *    {status: 'Draft'},
        * ]
+       * if created by you filter, follows different logic so if else statement to check
        */
       if (!isKeywordField && !hasAllStatusesFilterValue) {
         const fieldFilter = {};
         // or for field (eg dealType)
         fieldFilter.$or = [];
         filterValue.forEach((value) => {
-          fieldFilter.$or.push({
-            [fieldName]: value,
-          });
+          // if created by you then adding user id as compared to or statement
+          if (value === CONTENT_STRINGS.DASHBOARD_FILTERS.BESPOKE_FILTER_VALUES.DEALS.CREATED_BY_YOU) {
+            // delete or filter as not needed for created by you
+            delete fieldFilter.$or;
```

Comment:
are users able to sort based on `createdByYou` while having all other filters enabled?

Reply:
yes its working alongside all other filters